pxtsim/runtime.ts

```diff
@@ -34,16 +34,14 @@ namespace pxsim {
         }
 
         export function now(): number {
-            return Date.now();
```

Comment:
This is bad idea. performance.now() returns time since web navigation.

Keep a separate function, and maybe name it `perfNow()`.